pkg/arvo/sys/hoon.hoon

```diff
@@ -16793,7 +16811,27 @@
           ==
         ==
       ==
-    ::
+    ::  parses [a b c] or ==  a  b  c  ==
```

Comment:
Why does this having a leading `==`?  There's a precedent for a `==` that isn't at the end of the twig (`=:`), but none for circumfix `==`.

Reply:
Untrue. There is a rune that uses == I just... forget which one it is. I'll get back to you.

Reply:
~% possibly? That's still in a fundamentally close-paren capacity iirc, == meaning both ] and [ is just confusing

Reply:
Yes. ~%, thank you. In what way do those function as close-parens? the first one I mean?

Reply:
have a look at the ~% on ++ut

Reply:
Ah, yes.  It's still weird, but at least there's a precedent.

Reply:
~% really isn’t a good precedent. Let’s not make more runes like it.